src/dynarec/arm64/dynarec_arm64_00.c

```diff
@@ -2578,7 +2578,7 @@ uintptr_t dynarec64_00(dynarec_arm_t* dyn, uintptr_t addr, uintptr_t ip, int nin
                         STORE_XEMU_CALL(xRIP);
                         ADDx_U12(x3, xRIP, 8+8);    // expected return address
                         ADDx_U12(x1, xEmu, (uint32_t)offsetof(x64emu_t, ip)); // setup addr as &emu->ip
-                        CALL_(x64Int3, -1, x3);
```

Comment:
I am neatpeeking here, but x64Int3 is not only for Native Call, but also for guenuine Int 3 `CC` opcodes.

Reply:
indeed, how about changing it to `EmuInt3`?

Reply:
yeah, sounds good to me